src/components /AlertButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import { Link, NavLink } from "react-router-dom";
+import { BsExclamationDiamond } from "react-icons/bs";
+
+const AlertButton = (props) => {
+  const phoneNumber = [];
```

Comment:
This seems non-functional to me. I think it would be good to make it work with the user's phone number, even if you only have 1 user as of now.